### PR TITLE
[Fix] Fix unittest for Numclasshook when it is not used.

### PR DESCRIPTION
**Pull request by AronLin**
When `Numclasshook` is not used in the config, the unit test for this config still runs `_check_numclasscheckhook`.

I fix it by adding checking.

### CODE REVIEW

File: tests/test_runtime/test_config.py [hunk 1]
```diff
@@ -93,7 +92,16 @@ def test_config_build_detector():
         detector = build_detector(config_mod.model)
         assert detector is not None
 
-        _check_numclasscheckhook(detector, config_mod)
+        # Check whether NumClassCheckHook is used.
+        assert config_mod.custom_hooks is None or \
```
Comment (jshilong) [inline tests/test_runtime/test_config.py:96]: config_mod.get("custom_hooks", None)

**Review by jshilong — APPROVED** [review]
LGTM

File: tests/test_runtime/test_config.py (hunk 1 above)
Comment (ZwwWayne) [reply tests/test_runtime/test_config.py:96]: config_mod.get('custom_hooks', [])

File: tests/test_runtime/test_config.py [hunk 2]
```diff
@@ -93,7 +92,16 @@ def test_config_build_detector():
         detector = build_detector(config_mod.model)
         assert detector is not None
 
-        _check_numclasscheckhook(detector, config_mod)
+        # Check whether NumClassCheckHook is used.
+        custom_hooks = config_mod.get('custom_hooks', [])
+        assert custom_hooks is None or isinstance(custom_hooks, list)
```
Comment (ZwwWayne) [inline tests/test_runtime/test_config.py:97]: unnecessary assertion for None

Comment (AronLin) [reply tests/test_runtime/test_config.py:97]: `custom_hooks` is possibly set to `None`.

**Pull request merged**